packages/react/__tests__/responsive.test.tsx

```diff
@@ -31,6 +31,32 @@ describe('responsive', () => {
     expect(el.clientWidth).toBe(250);
   });
 
+  it('Should respect single step and ignore default width of 250', async function () {
+    const component = mount(
+      <ResponsiveHelper>
+        <AdvancedImage cldImg={cloudinaryImage} plugins={[responsive({steps: 251})]} />
+      </ResponsiveHelper>);
+
+    window.dispatchEvent(new Event('resize'));
+    clock.tick(100); // timeout for debounce
+
+    // Output is exactly 251, no matter what size the parent container here (Due to steps being defined as 251)
+    expect(component.html()).toContain('src="https://res.cloudinary.com/demo/image/upload/c_scale,w_251/sample"');
+  });
+
+  it('Should respect steps and ignore default width of 250', async function () {
+    const component = mount(
+      <ResponsiveHelper>
+        <AdvancedImage cldImg={cloudinaryImage} plugins={[responsive({steps: [10, 20, 30]})]} />
+      </ResponsiveHelper>);
+
+    window.dispatchEvent(new Event('resize'));
```

Comment:
What happens without a resize event?

Reply:
You're right, this is a mistake.
I've removed the events as the whole point is to show what happens before the resize event is fired. (The tests pass and it's working as intended, but the events should not be here).

Removed.